### PR TITLE
Release: 7.2.1

### PR DESCRIPTION
**Pull request by gardner48**
SUNDIALS Release v7.2.1

### CODE REVIEW

**Review by drreynolds — DISMISSED** [review]

**Review by balos1 — APPROVED** [review]
I added a note to the changelog about removing unit tests from the examples directory.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # SUNDIALS Changelog
 
-## Changes to SUNDIALS in release X.Y.Z
-
-### Major Features
+## Changes to SUNDIALS in release 7.2.1
 
 ### New Features and Enhancements
 
+Unit tests were separated from examples. To that end, the follow directories
```
Comment (drreynolds) [inline CHANGELOG.md:7]:
```suggestion
Unit tests were separated from examples. To that end, the following directories 
```

File: doc/shared/RecentChanges.rst [hunk 2]
```diff
@@ -1,12 +1,12 @@
-**Major Features**
+** New Features and Enhancements **
 
-**New Features and Enhancements**
+Unit tests were separated from examples. To that end, the follow directories
```
Comment (drreynolds) [inline doc/shared/RecentChanges.rst:3]:
```suggestion
Unit tests were separated from examples. To that end, the following directories 
```

**Review by drreynolds — COMMENTED** [review]
A couple of typos, but otherwise looks good.

**Review by drreynolds — APPROVED** [review]

**Pull request merged**